nasa-gibs/worldview
# Clean up of unused geostationary modal

## Description

Fixes #3445 .

- [x] Removed the unused `geostationary.gif`
- [x] Commented out `geostationary-modal.css`
- [x] Commented out and removed portions of `geostationary-modal.js` to leave as a bare bones component

## How To Test

- For enhancements and features: What is the expected functionality being added/modified? How can a reviewer verify this?

1. Look through geostationary modal files to see that the `geostationary-modal.gif` has been removed,  and that `geostationary-modal.css` and `geostationary-modal.js` have been commented out


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview
